docs/swagger/v1.json

```diff
@@ -1,710 +1,975 @@
 {
-    "openapi": "3.0.1",
-    "info": {
-      "title": "Azure SignalR Service REST API",
-      "version": "v1"
+  "openapi": "3.0.1",
+  "info": {
+    "title": "Azure SignalR Service REST API",
+    "version": "v1"
+  },
+  "paths": {
+    "/api/v1/health": {
+      "head": {
+        "tags": [
+          "general"
+        ],
+        "summary": "Get service health status.",
+        "operationId": "HealthApi_Head_Index",
+        "responses": {
+          "200": {
+            "description": "The service is healthy"
+          },
+          "503": {
```

Comment:
also 502? https://github.com/Azure/azure-signalr/blob/dev/src/Microsoft.Azure.SignalR.Management/ServiceManager.cs#L168

Reply:
swagger describes the service behavior, it does not include cases when service/network is down

Reply:
yeah you are right we missed several expected status code

Reply:
I feel we don't need to list 5xx code, every API can encounter 5xx. And for the client side, it should consider all 5xx code as `isHealthy = false`

Reference: https://stackoverflow.com/a/58277444/687394